src/Templates/AllWidget/AllWidget.js

```diff
@@ -477,37 +477,41 @@ export class AllWidget {
       },
       this.view
     );
+    $3dTilesLayer.haveCityObject = true;
 
     let material;
     if (layer['pc_size']) {
+      $3dTilesLayer.haveCityObject = false;
       material = new THREE.PointsMaterial({
         size: layer['pc_size'],
-        vertexColors: true,
+        vertexColors: THREE.VertexColors
       });
+      $3dTilesLayer.overrideMaterials = material;
+      $3dTilesLayer.material = material;
     }
 
-    $3dTilesLayer.overrideMaterials = material;
-    $3dTilesLayer.material = material;
-
     const $3DTilesManager = new TilesManager(
       this.view,
       $3dTilesLayer
     );
-    let color = 0xffffff;
-    if (layer['color']) {
-      color = parseInt(layer['color']);
-    }
-    $3DTilesManager.registerStyle('default', {
-      materialProps: { opacity: 1, color: color },
-    });
-
-    $3DTilesManager.addEventListener(
-      TilesManager.EVENT_TILE_LOADED,
-      function (event) {
-        $3DTilesManager.setStyleToTileset('default');
-        $3DTilesManager.applyStyles();
+    
+    if(!layer['pc_size']){
```

Comment:
Is it obvious to what layer['pc_size'] corresponds !?
Maybe a little comment on what the code is achieving in this block ?

Reply:
Done